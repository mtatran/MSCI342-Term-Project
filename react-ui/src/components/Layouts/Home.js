import React, { Component, Fragment } from "react";

import Title from "../Title"
import AppointmentList from "../AppointmentList/AppointmentList";

class Home extends Component {
  render() {
    return (
      <Fragment>
        <Title name="Welcome"/>
        <AppointmentList user={this.props} />
      </Fragment>
    );
  }
}
export default Home;
