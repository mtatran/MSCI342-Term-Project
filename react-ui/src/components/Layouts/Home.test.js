import React from "react";
import { create } from "react-test-renderer";
import Home from './Home';

describe('Home Snapshot', () => {
  test('snapshot renders', () => {
    const component = create(<Home />);
    expect(component.toJSON()).toMatchSnapshot();
  });

});