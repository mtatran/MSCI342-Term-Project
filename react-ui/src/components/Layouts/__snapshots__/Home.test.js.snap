// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot snapshot renders 1`] = `
Array [
  <div>
    <p
      className="Title-Style"
    >
      Welcome
    </p>
  </div>,
  <h3>
    No Upcoming Appointments!
  </h3>,
]
`;
